app/views/partials/costEstimation.scala.html

```diff
@@ -14,21 +14,43 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.cataloguefrontend.service.CostEstimationService.CostEstimation
+@import uk.gov.hmrc.cataloguefrontend.service.CostEstimationService.ServiceCostEstimate
 @import uk.gov.hmrc.cataloguefrontend.util.CurrencyFormatter.formatGbp
 
-@(costEstimation: CostEstimation)
+@(serviceCostEstimate: ServiceCostEstimate, serviceCostEstimateConfig: ServiceCostEstimate.Config)
 <div id="cost-estimation">
     <div class="board">
         <h3 class="board__heading">Estimated Cost</h3>
         <div class="board__body">
-            <p><b>@formatGbp(costEstimation.yearlyCostGbp)</b> per year.</p>
-            <p>This is based on a usage of @costEstimation.totalSlots slots across all environments, where
+            <p><b>@formatGbp(serviceCostEstimate.summary.totalYearlyCostGbp)</b> per year.</p>
+            <p>This is based on a usage of @serviceCostEstimate.summary.totalSlots slots across all environments, where
                <em data-toggle="tooltip"
-                     title="We arrive at this figure, very roughly, by dividing MDTP's yearly AWS spend of £5.4M by the
-                            total number of slots used by all services across all environments">
-                   each slot is estimated to cost £650.00 per year
+                     title="We arrive at this figure, very roughly, by dividing MDTP's yearly AWS spend of
+                            @serviceCostEstimateConfig.totalAwsCostPerYear by the total number of slots used by all
+                            services across all environments">
+                   each slot is estimated to cost @formatGbp(serviceCostEstimateConfig.slotCostPerYear) per year
                </em>.</p>
+            <div id="chart" />
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+<script type="text/javascript">
+      google.charts.load("current", {packages:["corechart"]});
+      google.charts.setOnLoadCallback(drawChart);
+
+      function drawChart() {
+        var data = google.visualization.arrayToDataTable([
+          ['Environment',   'Estimated Cost'],
+          @for(e <- serviceCostEstimate.forEnvironments) {
+            ['@e.environment.asString', { v: @e.yearlyCostGbp, f: "@formatGbp(e.yearlyCostGbp) (slots = @e.slots)" } ],
```

Comment:
Is this super gross? Should I create a view model and associated JSON format to do this properly?